cmd/frontend/internal/app/jscontext/jscontext.go

```diff
@@ -143,8 +149,10 @@ func NewJSContextFromRequest(req *http.Request, db database.DB) JSContext {
 	}
 
 	var githubAppCloudSlug string
+	var githubAppClientID string
```

Comment:
```suggestion
	var githubAppCloudClientID string
```

I think having "Cloud" in the name is important for a) being consistent with every other variable naming, b) indicating this is the GitHub App specifically for Sourcegraph Cloud. "githubApp" alone is a very generic name IMO. WDYT?

Reply:
100% agree, didn't realise that "Cloud" was present in the other variable names